Library/Formula/sgfutils.rb

```diff
@@ -0,0 +1,68 @@
+class Sgfutils < Formula
+  desc "collection of command-line utilities to work with SGF files"
+  homepage "http://homepages.cwi.nl/~aeb/go/sgfutils/html/sgfutils.html"
+  url "http://homepages.cwi.nl/~aeb/go/sgfutils/sgfutils-0.22.tgz"
+  sha256 "4ad83be91ac3a52dd8bd0766bfe403499430da96e680c5aea2f5381ba0b34923"
+
+  depends_on "openssl" => :build
+
+  # Makefile tweaks for OS X. The corresponding patch was sent to the
+  # main author of sgfutils on 2016-03-17 for potential upstream
+  # inclusion.
+  patch :DATA
+
+  def install
+    bin_files = %w[
+      ngf2sgf nip2sgf nk2sgf sgf sgfcharset sgfcheck sgfcmp sgfdb sgfdbinfo
+      sgfinfo sgfmerge sgfsplit sgfstrip sgftf sgftopng sgfvarsplit sgfx ugi2sgf
+    ]
+    openssl_includes = Formula["openssl"].opt_prefix + "include"
+    cflags = "-Wall -O3 -I#{openssl_includes}"
+    system "make", "all", "CFLAGS='#{cflags}'"
+    bin_files.each { |file| bin.install file }
+  end
+
+  test do
+    (testpath/"temp.sgf").write "(;SZ[2];B[aa];W[ab];B[bb])"
+    actual = shell_output "#{bin}/sgftf -rot90 temp.sgf"
```

Comment:
Not specific to this. There's a [Cookbook](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md) that covers an awful lot, but for little things like this checking the other formulae tends to be the first choice of most contributors.